pkg/build/api/v1/types.go

```diff
@@ -303,6 +303,9 @@ type BuildStrategy struct {
 
 	// CustomStrategy holds the parameters to the Custom build strategy
 	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty" description:"holds parameters to the Custom build strategy"`
+
+	// ExternalStrategy holds the parameters to the External build strategy
+	ExternalStrategy *ExternalBuildStrategy `json:"externalStrategy,omitempty" description:"holds parameters to the External build strategy"`
```

Comment:
@mfojtik fabric knows how to interrogate a jenkins workflow job definition and visualize it.  that's what will come to openshift.  The buildconfig will just point to that workflow job.  (that's one scenario.  another is that that buildconfig actually contains enough metadata on its own to define a workflow, without a jenkinsfile).

but no, there will not be linked buildconfigurations, not all steps will be represented as buildconfiguration objects (eg a human approval step)